qt/python/mantidqt/widgets/plotconfigdialog/curvestabwidget/presenter.py

```diff
@@ -242,8 +228,22 @@ def set_errorbars_tab_enabled(self):
         enable_errorbars = curve_has_errors(self.get_selected_curve())
         self.view.set_errorbars_tab_enabled(enable_errorbars)
 
-    def update_view(self):
+    def on_axes_index_changed(self):
+        self.update_view(update_axes=False)
+
+    def on_curves_index_changed(self):
+        self.update_view(update_axes=False, update_curves=False)
+
+    def update_view(self, update_axes=True, update_curves=True):
         """Update the view with the selected curve's properties"""
```

Comment:
Can we update this doc string now, it should mention:
- What each param does and what they update
- If possible what properties (though if it's all of them, feel free to just leave it at properties)